vcd/resource_vcd_vapp_vm.go

```diff
@@ -1158,6 +1162,18 @@ func updateStateOfAttachedDisks(d *schema.ResourceData, vm govcd.VM, vdc *govcd.
 	return d.Set("disk", transformed)
 }
 
+func filterIndependentDiskFromVmDisks(vm govcd.VM, diskHref string) (*types.DiskSettings, error) {
```

Comment:
The name "filterIndependentDiskFromVmDisks" can be interpreted two ways: either you filter out or filter in. What about naming "findIndependentDiskInVmDisks" or "getIndependentDiskFromVmDisks"?

Reply:
changed